src/components/YourModule.js

```diff
@@ -52,58 +51,28 @@ export default class YourModule extends React.Component {
   }
 
   bet(outcome) {
-    //Gnosis.create(
-    //  {ethereum: this.props.xdaiweb3.provider}
-    //).then(async (result) => {
-      console.log("BETTING...")
-      //const market =  result.contracts.Market.at(this.state.address);
       const cost = Gnosis.calcLMSROutcomeTokenCount(
           this.state.outcomeTokensSold,
           1000e18,
           outcome,
           this.state.amount * 1e18,
           0
       );
-      console.log("CALL buy(",outcome,",",cost.toNumber(),",",10 * 1e18,")")
-      //console.log("market:",market)
```

Comment:
Removed while trying to make the past tx, please le me know if you want them so I commit them back